dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/oracle/AbstractOracleConnector.java

```diff
@@ -73,6 +73,10 @@
 })
 public abstract class AbstractOracleConnector extends AbstractJdbcConnector {
 
+  /* This argument is required to improve performance of retrieving LONG columns */
```

Comment:
This flag can improve performance, but it may also cause issues, if the data is too big.
https://docs.oracle.com/en/database/oracle/oracle-database/21/jajdb/oracle/jdbc/OracleConnection.html#CONNECTION_PROPERTY_USE_FETCH_SIZE_WITH_LONG_COLUMN

I would suggest making it possible to turn off this parameter, so that the user can choose to wait longer and retrieve the data without errors if enabling the flag causes issues.